examples/combobox/combobox-datepicker.html

```diff
@@ -0,0 +1,781 @@
+                <td tabindex="-1" data-date="2020-02-24">24</td>
+                <td tabindex="-1" data-date="2020-02-25">25</td>
+                <td tabindex="-1" data-date="2020-02-26">26</td>
+                <td tabindex="-1" data-date="2020-02-27">27</td>
+                <td tabindex="-1" data-date="2020-02-28">28</td>
+                <td tabindex="-1" data-date="2020-02-29">29</td>
+              </tr>
+              <tr>
+                <td tabindex="-1" data-date="2020-02-30">30</td>
+                <td tabindex="-1" data-date="2020-02-31">31</td>
+                <td class="disabled" tabindex="-1"></td>
+                <td class="disabled" tabindex="-1"></td>
+                <td class="disabled" tabindex="-1"></td>
+                <td class="disabled" tabindex="-1"></td>
+                <td class="disabled" tabindex="-1"></td>
+              </tr>
+            </tbody>
+          </table>
+
+          <div class="dialog-ok-cancel-group">
+            <button class="dialog-button" value="cancel">Cancel</button>
+            <button class="dialog-button" value="ok">OK</button>
+          </div>
+
+          <div class="dialog-message" aria-live="polite"></div>
+
+        </div>
+      </div>
+    </div>
+    <div role="separator" id="ex_end_sep" aria-labelledby="ex_end_sep ex_label" aria-label="End of"></div>
+  </section>
+
+  <section>
+    <h2>Accessibility Features</h2>
+    <p>There are three main features of the combobox date picker example:</p>
+    <ul>
+      <li>Textbox: Holds the value of the selected date and can open the date picker dialog box.</li>
+      <li>Choose Date Button: Opens the datepicker dialog box.</li>
+      <li>Date Picker Dialog Box: Displays a grid of dates for the user to select from and provides additional buttons to change the month and year of dates shown in the grid.</li>
+    </ul>
+
+    <p>The <code>combobox</code>, <code>textbox</code> and <code>button</code> all have the same accessible name.</p>
+
+
+    <h3>Date Textbox</h3>
+
+    <ul>
+      <li>Contains the date value.</li>
+      <li>Opens and closes the date picker dialog box through keyboard, click and touch events.</li>
+      <li>A down arrow icon in the <code>textbox</code> provides a visual affordance that the <kbd>down arrow</kbd> key opens the date picker dialog box.</li>
+      <li>A description provides information on the date format to assistive technologies.</li>
+    </ul>
+
+    <h3>Choose Date Button</h3>
+
+    <ul>
+      <li>The down arrow icon is a button to open and close the date picker dialog box through click events.</li>
+      <li>The arrow points down when the dialog box is closed and it points up when the dialog box is open.</li>
+      <li>Excluded from the <kbd>tab</kbd> sequence of the page (eg. <code>tabindex=-1</code>), since the user can use the <kbd>down arrow</kbd> or <kbd>enter</kbd> key to open the date picker dialog box from the <code>textbox</code>.</li>
```

Comment:
use e.g. instead of eg.

```suggestion
      <li>Excluded from the <kbd>tab</kbd> sequence of the page (e.g. <code>tabindex=-1</code>), since the user can use the <kbd>down arrow</kbd> or <kbd>enter</kbd> key to open the date picker dialog box from the <code>textbox</code>.</li>
```